pkg/terra/contract_transmitter.go

```diff
@@ -55,7 +56,7 @@ func (ct *ContractTransmitter) Transmit(
 	}
 	msgStruct.Transmit.Report = []byte(report)
 	for _, sig := range sigs {
-		msgStruct.Transmit.Signatures = append(msgStruct.Transmit.Signatures, sig.Signature)
+		msgStruct.Transmit.Signatures = append(msgStruct.Transmit.Signatures, []byte(base64.StdEncoding.EncodeToString(sig.Signature)))
```

Comment:
Another line here would help readability:

```suggestion
                sigBase64 := []byte(base64.StdEncoding.EncodeToString(sig.Signature))
		msgStruct.Transmit.Signatures = append(msgStruct.Transmit.Signatures, sigBase64)
```